Marlin/Marlin_main.cpp

```diff
@@ -1644,7 +1639,7 @@ static void clean_up_after_endstop_or_probe_move() {
 
     #if ENABLED(DELTA)
 
-      feedrate = XY_PROBE_FEEDRATE;
+      feedrate = (feed_rate != 0.0) ? feed_rate : XY_PROBE_FEEDRATE;
```

Comment:
I have seen that too. As far it remember it was to was to tell the planner to begin the next move with zero sped. No - this is completely limited to the do_blocking move() family.